config/prod.exs

```diff
@@ -27,13 +27,15 @@ config :castle, :redis, Castle.Redis.Api
 # Scheduled jobs
 config :castle, Castle.Scheduler,
   jobs: [
-    {"* * * * *",     {Feeder.Sync,         :run, [["--lock"]]}},
-    {"15,45 * * * *", {Rollup.Hourly,       :run, [["--lock"]]}},
-    {"15 6 * * *",    {Rollup.Monthly,      :run, [["--lock"]]}},
-    {"20,50 * * * *", {Rollup.Geocountries, :run, [["--lock"]]}},
-    {"25,55 * * * *", {Rollup.Geometros,    :run, [["--lock"]]}},
-    {"30,0 * * * *",  {Rollup.Geosubdivs,   :run, [["--lock"]]}},
-    {"35,5 * * * *",  {Rollup.Agents,       :run, [["--lock"]]}},
+    {"* * * * *",   {Feeder.Sync,     :run, [["--lock"]]}},
+    {"*/5 8 * * *", {Postgres.Vacuum, :run, [["--lock"]]}},
```

Comment:
So with 12 five minute periods to run these vacuums, this should converge to having all the tables in castle vacuumed nicely.